traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -142,6 +159,36 @@ func SortTestRegions(t *testing.T) {
 	}
 }
 
+func SortTestRegionsDesc(t *testing.T) {
+
+	var header http.Header
+	respAsc, _, err1 := TOSession.GetRegions(nil, header)
+	params := url.Values{}
+	params.Set("sortOrder", "desc")
+	respDesc, _, err2 := TOSession.GetRegions(params, header)
+
+	if err1 != nil {
+		t.Errorf("Expected no error, but got error in Regions Ascending %v", err1)
+	}
+	if err2 != nil {
+		t.Errorf("Expected no error, but got error in Regions Descending %v", err2)
+	}
+
+	if len(respAsc) > 0 && len(respDesc) > 0 {
+		// reverse the descending-sorted response and compare it to the ascending-sorted one
+		for start, end := 0, len(respDesc)-1; start < end; start, end = start+1, end-1 {
+			respDesc[start], respDesc[end] = respDesc[end], respDesc[start]
+		}
+		if respDesc[0].Name != "" && respAsc[0].Name != "" {
+			if !reflect.DeepEqual(respDesc[0].Name, respAsc[0].Name) {
+				t.Errorf("Regions responses are not equal after reversal: %s - %s", *&respDesc[0].Name, *&respAsc[0].Name)
```

Comment:
nit but `*&` is a no-op; `*&respDesc.Name` is the same as just `respDesc.Name`

Reply:
Fixed.,